src/services.py

```diff
@@ -48,22 +49,27 @@ def __init__(self, config, hasher=argon2):
         self.config = config
         self.hasher = hasher
 
-    @validate_input(CREATE_USER_SCHEMA)
-    def create(self, email, password, full_name):
+    def create(self, email, display_image, full_name, user_id):
         with session_scope() as session:
-            hashed_password = self.hasher.hash(password)
-            user = User(
-                email=email,
-                full_name=full_name,
-                hashed_password=hashed_password,
-                can_buy=False,
-                can_sell=False,
-            )
-            session.add(user)
-            session.commit()
+            user = session.query(User).filter_by(user_id=user_id).one_or_none()
+            if user is None:
+                user = User(
+                    email=email,
+                    full_name=full_name,
+                    display_image=display_image,
+                    provider="linkedin",
+                    can_buy=False,
+                    can_sell=False,
+                    user_id=user_id,
+                )
+                session.add(user)
+            else:
+                user.display_image = display_image
```

Comment:
If the user exists, why still change these stuff?

Reply:
Because if the user updates their email, or change user profile, then when they login, these fields get updated.

Is that alright?